test/psxy/line_geo.sh

```diff
@@ -2,8 +2,9 @@
 # Test strike/dip symbol from Jose for consistency across quadrants
 # This version is a Mercator plot and azimuths are given.
 ps=line_geo.ps
-# Must temporarily change GMT_USERDIR to the gallery documentation dir
-export GMT_USERDIR=`gmt --show-sharedir`/../doc/rst/source/users-contrib-symbols/geology
+# Must copy the specific symbol file from the documentation to here.
+DIR=`gmt --show-sharedir`/../doc/rst/source/users-contrib-symbols/geology
+cp -f ${DIR}/geo-lineation.def .
```

Comment:
It would be better to remove the def file at the end of the script.

Reply:
Done